### ISSUE TITLE
dpt rp1 stop on welcome after step 3

### ISSUE THREAD

**Issue by yogi08** [HappyZ/dpt-tools#118]
Hi, after step by step in your tutorial, my DPT stay on welcome screen and could not enter in system. How to recover it. I try to reconnect it to digital paper app but failed.

Any suggestion?

**Comment by 6bigfire** [comment]
How do you operate step by step?
fwupdatered in debug mode?

**Comment by yogi08** [comment]
The steps is :
python dpt-tools.py ip
fw 
yes
fw.pkg

fw
yes
FwUpdater.pkg


and the dpt enter in "welcome " screen and never exit

**Comment by 6bigfire** [comment]
which “FwUpdater.pkg”?

**Comment by yogi08** [comment]
This one
\fw_updater_packer_unpacker\pkg_example\flashable_mod_boot_img_1.6.00.15110

**Comment by yogi08** [comment]
how can I flash back to the official system? HappyZ master.

**Comment by 6bigfire** [comment]
if there have no error before, you should wait a few minutes still the welcome disappear.
Now you can try hold the home button in poweroff status, and push the power button to power on The device, when the yellow LED flashing, hold the home button for a few seconds and release. when the black square is appeared, you can try to link the device in diagnosis mode, and you can reflush it.

**Comment by HappyZ** [comment]
which version was your original system? it will stuck if yours was 1.4xx but you flashed 1.6

**Comment by HappyZ** [comment]
just flash the right version to it, it then won't stuck there

**Comment by yogi08** [comment]
@6bigfire @HappyZ  I can't enter into the diagnosis mode. I did the following steps.
1. reset it to power off
2. press home + power button until the yellow LED flashing, then release the power button with home button still on
3. a few seconds later, there were a few squares at the left right corner and a big square at the center
4. a few seconds later, I release the home button. But the squares are never dispear. The yellow and red LED flashing alternatively.

Is there any other method to reboot the device?

**Comment by HappyZ** [comment]
Seeing the square at the center means you entered diagnosis mode, you need to connect your DPT to computer and use the tool to make a connection (by entering the correct serial port).

**Comment by HappyZ** [comment]
Once you connected via the tool, just type in `install-pkg` and follow the instruction.

**Comment by yogi08** [comment]
@HappyZ  Thanks, Happy.

Through this way, my DP enter into a screen with "updating... please wait", then reboot automatically and enter into "welcome" screen. The "welcome" screen lasting more than half an hour now. Is it normal? how long usually it lasting?

**Comment by yogi08** [comment]
She returns! really thanks for your advice!

**Comment by tangshi141** [comment]
@HappyZ, I have the same situation as yogi08. But when I following the solution and type “fw” or “diagnosis” the console automatically restarts. Can you further explain how can I connect with my serial port with the dpt-tools.py?

**Comment by HappyZ** [comment]
@tangshi141  please start a new issue. What do you mean by console automatically restarts? If you see the black squares, use `python dpt-tools.py --diagnosis` directly to enter. The serial port for Windows should be something like COMx where x is a number.  For Mac it would be /dev/tty.usbserialxxx, and for Linux it should be /dev/ttyUSBx
